apps/cowswap-frontend/src/modules/trade/pure/TradeConfirmation/CountDown.tsx

```diff
@@ -1,15 +1,35 @@
 import { useEffect, useState } from 'react'
 
+import ms from 'ms.macro'
+
 import { QuoteCountdownWrapper } from './styled'
 
+const ONE_SEC = ms`1s`
+
 interface CountdownComponentProps {
-  nextUpdateAt: number
+  refreshInterval: number | undefined
 }
 
-export const QuoteCountdown = ({ nextUpdateAt }: CountdownComponentProps) => {
+export const QuoteCountdown = ({ refreshInterval }: CountdownComponentProps) => {
```

Comment:
No real changes, just refactored a bit in order to reduce rerenders in the parent component:
https://github.com/cowprotocol/cowswap/pull/5485/commits/c4535536953b036d7d2d597039954cc534fc607c